ako-infra/ingestion/avi_infra_controller.go

```diff
@@ -103,7 +103,9 @@ func (a *AviControllerInfra) DeriveCloudNameAndSEGroupTmpl(tz string) (error, st
 		}
 		vtype := *cloud.Vtype
 		if vtype == lib.CLOUD_NSXT && cloud.NsxtConfiguration != nil {
-			if cloud.NsxtConfiguration.TransportZone != nil && *cloud.NsxtConfiguration.TransportZone == tz {
+			//cloud.NsxtConfiguration.TransportZone field is deprecated in 20.1.5
```

Comment:
@abhishekbsingh

Reply:
AKO-Infra changes have already been covered in the PR https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pull/955/files, we can remove these changes from the PR here